Landing CV/index.html

```diff
@@ -0,0 +1,305 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Landing CV</title>
+    <link rel="stylesheet" href="styles/slyles.css">
+</head>
+<body>
+<div class="container">
+    <!--Header-->
+    <header class="header">
+        <nav class="nav">
+            <a href="/">About</a>
+            <a href="/">Skills</a>
+            <a href="/">Experience</a>
+            <a href="/">Projects</a>
+            <a href="/">Education</a>
+            <a href="/">Languages</a>
+            <a href="/">Contact me</a>
+        </nav>
+    </header>
+    <!--About me-->
+    <section class="about_me_section">
+        <div class="about_me">
+            <h2>Sergei Alyoshin</h2>
+            <h2>Front-end developer</h2>
+            <div class="about_me_text">
+                <p>Hi, I am Sergei Alyoshin and I am Junior 
+                    Front-end <br> developer. <br><br>
+                </p>
+                <p>I have completed Front-end courses at IT 
+                    School Intellect <br> Fox and have basic knowledge 
+                    of HTML, CSS, JavaScript,<br> React and Redux.</p>
+            </div>
+            <form class="about_me_bottoms" action="">
+                <input class="button" type="button" value="Download CV">
+                <input class="button" type="button" value="Contact me">
+            </form>
+        </div>
+        <img class="photo" src="img/photo.svg" alt="my photo">
+    </section>
+    <!--Skills-->
+    <section class="skills_section">
+        <h2 class="skills_name">Skills</h2>
+        <div class="skills_panel">
+            <div class="skills">
+                <img src="img/HTML5_Logo 12.svg" alt="html">
+                <div class="skills_info">
+                    <span>HTML</span>
+                    <div class="stars">
```

Comment:
add paddings and margins
![image](https://user-images.githubusercontent.com/20189313/129493280-4619c18b-9245-4682-b3d5-def35fc9f47f.png)